api/jobs/handlers.py

```diff
@@ -0,0 +1,105 @@
+"""
+API request handlers for the jobs module
+"""
+
+from ..dao.containerutil import create_filereference_from_dictionary, create_containerreference_from_dictionary
+
+from .. import base
+from .. import config
+
+from .jobs import Job
+from .queue import Queue
+
+log = config.log
+
+
+class JobsHandler(base.RequestHandler):
+
+    """Provide /jobs API routes."""
+
+    def get(self):
+        """
+        List all jobs.
+        """
+        if not self.superuser_request:
+            self.abort(403, 'Request requires superuser')
+
+        return list(config.db.jobs.find())
```

Comment:
Agreed - *nobody* should consume this endpoint :)

I would be happy to remove it; this is a pre-existing endpoint originally added to match the convention of other endpoints that dump entire tables. Since this PR aims to have no exported changes, I'd like to leave it for now.